### PR TITLE
Updated the website to look better

### PR DESCRIPTION
**Pull request by o3LL**
Website was _meh_, now website is **wow**!

### CODE REVIEW

**Comment by o3LL** [comment]
Preview of the modifications for the most lazy
website preview

File: server/static/js/chaos.js [hunk 1]
```diff
@@ -52,3 +51,18 @@ setInterval(function() {
...
+    if (n === k.length) {
+      document.body.className = 'transform';
+      setTimeout(function(){ document.body.className = ''; },15000);
+      n = 0;
+      return false;
+    }
+  } else {
+        n = 0;
+  }
+});
```
Comment (phil-r) [inline server/static/js/chaos.js:68]: What is this magic above? O_O

Comment (o3LL) [reply server/static/js/chaos.js:68]: As describe in the commit message, this is not even an easter egg 
⬆️ ⬆️ ⬇️ ⬇️ ⬅️ ➡️ ⬅️ ➡️ 🅱️ 🅰️

Comment (phil-r) [reply server/static/js/chaos.js:68]: OMG, get all my upvotes! Why I haven't figured it out myself 🤦‍♂️

**Review by bengjerstad — APPROVED** [review]

**Pull request merged**